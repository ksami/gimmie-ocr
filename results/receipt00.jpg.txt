Actual | Predicted | OCR Conf
FAI-QQFQEEEE IX-T-RA | (FAIRPRICE XTRA, 53) | 30.195122
. HEX HYPERHART | (NEX HYPERMART, 85) | 62.586277
23 SWW-BWHH 1303-42 | (BASIL 30G, 40) | 51.669899
SIWR 556083 | (BASIL 30G, 40) | 66.608002
----M TEE-6834 4947/4948 -. . . | (REG, 30) | 57.571758
REG M5 RES 1.1.191.1 | (REG, 90) | 72.224892
SST PM 34-004573-0 | (GST, 60) | 64.189545
. SGD | (SGD, 100) | 77.151794
LI-TNLQCHGGN EGG 513 G ZAU | (SENGCHOON EGG 55 G, 55) | 63.525803
GD SN ENR MT 8609. 1.20 | (GD S/V ENR WHT 600G, 67) | 67.557640
N-FRS SGYAMK ALMNL - -- 2.05 | (N-FRS SOYAMK ALMD1L, 72) | 78.310127
NJRS SGYAHK ALHNL 2.05 | (SGD, 60) | 64.543274
H5 SPINACH 2509 1.55 | (FP SPINACH 250G, 69) | 65.108940
F9 PF SATAY-BEEFSOO 5.75 | (FP PF SATAY-BEEF500, 74) | 70.138184
NORMAL PRICE 6.06 | (NORMAL PRICE, 95) | 76.411453
BELHGNTE BMMS-BAG 0.99 | (GST, 60) | 65.156067
LINK CARG 3501600000168019 | (LINK CARD, 86) | 67.448853
2X 0.32-   2 | (FP PF SATAY-BEEF500, 38) | 76.054977
NSBY FR S/HILK A -.   0.65- | (NSOY FR S/MILK A, 86) | 77.602249
SUBTOTAL .- --. 15.34 | (SUBTOTAL, 90) | 79.389961
TOTAL . -.  15.34 | (TOTAL, 90) | 75.638626
CM H8 I11360UG1   0-45 . | (CASH, 45) | 77.646866
RUUHDING A -   .. . .- .. | (NSOY FR S/MILK A, 86) | 68.513412
WWEJW 12.23 | (RAD SHRED-MOZZ 250G, 32) | 46.288445
YE . .... - 14.85N | (YOPLT L/YOG B/B 125G, 42) | 56.894409
