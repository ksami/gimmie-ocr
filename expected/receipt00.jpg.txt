SENGCHOON EGG 55 G 2.40
GD S/V ENR WHT 600G 1.20
N-FRS SOYAMK ALMD1L 2.05
N-FRS SOYAMK ALMD1L 2.05
FP SPINACH 250G 2.05
FP PF SATAY-BEEF500 1.55
!NORMAL PRICE 6.06
DELMONTE BANANAS-BAG 5.75
!LINK CARD 8301000000168919
!2X 0.32-
NSOY FR S/MILK A 0.65-
!SUBTOTAL 15.34
!TOTAL 15.34
!Card HB discount 0.46
!ROUNDING ADJUSTMENTS 0.03-
!BALANCE DUE 14.85
!VISA 14.85